guides/common/attributes.adoc

```diff
@@ -19,16 +23,16 @@
 :RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-beta-rpms
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
```

Comment:
It feels weird to have this. It's not used anywhere and 6.3 is EOL. Should this at least be moved to a Satellite attributes file that's only included in downstream builds? The moving part refers to all these things, all the way up to TargetVersion.

Reply:
Agreed, let me move all those RepoRHEL7* attributes into attributes-satellite.adoc alongside with SatelliteSub. These should not be used upstream at all.